frontend/public/components/modals/tags.tsx

```diff
@@ -76,18 +81,6 @@ export const annotationsModal = createModalLauncher((props: AnnotationsModalProp
   />
 ));
 
-type TagsModalProps = {
-  tags?: { [key: string]: string };
-  path: string;
-  titleKey: string;
-  kind: K8sKind;
-  resource: K8sResourceKind;
-  inProgress: boolean;
-  errorMessage: string;
-  cancel?: () => void;
-  close?: () => void;
-} & HandlePromiseProps;
-
-type AnnotationsModalProps = Omit<TagsModalProps, 'path' | 'tags'>;
+export { TagsModalProps, AnnotationsModalProps };
```

Comment:
Note that both `TagsModalProps` and `AnnotationsModalProps` types were not exported.

Therefore, we don't need to re-export them like above.